src/app/nation/nation.component.ts

```diff
@@ -24,7 +24,7 @@ export class NationComponent implements OnInit {
   message = '';
   nations = [];
   nationForm: FormGroup;
-  searchText='';
+  search_text='';
```

Comment:
Camel case is more standard for JavaScript, so please revert this variable to `searchText`